sources/packages/api/src-sql/StudentScholasticStandings/Remove-scholastic-standing-approval-fields.sql

```diff
@@ -0,0 +1,12 @@
+-- Drop approval realted fields.
```

Comment:
Do we still need `the scholastic_standing_status`? Since we are removing the other ones this one can also be removed.

Reply:
I believe in UI data-table(History) we can hardcode the status based on the type.